dspace-api/pom.xml

```diff
@@ -628,13 +628,13 @@
             <artifactId>aws-java-sdk-s3</artifactId>
             <version>1.10.50</version>
         </dependency>
-
-        <!-- For ORCID v2 integration -->
+        
         <dependency>
-            <groupId>org.dspace</groupId>
-            <artifactId>orcid-jaxb-api</artifactId>
-            <version>2.1.0</version>
-        </dependency>
+			<groupId>org.dspace</groupId>
+			<artifactId>orcid-jaxb-api</artifactId>
+			<version>3.0.0-SNAPSHOT</version>
+		</dependency>
```

Comment:
The spacing here is messed up. It looks like you are using tabs instead of spaces. Could you correct the alignment here?